plugins/kotlin/frontend-independent/tests/test/org/jetbrains/kotlin/test/utils/IgnoreTests.kt

```diff
@@ -282,6 +290,7 @@ object IgnoreTests {
                 firTestFile.delete()
                 originalTestFile.prependFirIdentical()
                 additionalFileToMarkFirIdentical?.prependFirIdentical()
+                if (additionalFileToDeleteIfIdentical?.exists() == true) additionalFileToDeleteIfIdentical.delete()
```

Comment:
I believe you can just call `?.delete()` here without check

Reply:
Thanks! Just tried and indeed it's safe to delete an nonexistent file.